sesame/src/main/scala/org/w3/banana/sesame/io/SesameRDFWriter.scala

```diff
@@ -10,14 +11,14 @@ class SesameRDFWriter[T](implicit
   sesameSyntax: SesameSyntax[T]
 ) extends RDFWriter[Sesame, Try, T] {
 
-  def write(graph: Sesame#Graph, os: OutputStream, base: String): Try[Unit] = Try {
+  def write(graph: Sesame#Graph, os: OutputStream, base: String, prefixes: Prefix[Sesame]*): Try[Unit] = Try {
```

Comment:
I implemented namespace-support also for Sesame. Had to make a change to how we print relative URIs (not supported in the version of Sesame linked to banana-rdf right now at least), hopefully I didn't brake anything.

Reply:
If you're worried it may be useful to write a test which takes a graph with the prefixes, writes them out, then parses the result in again (after the tests you just made) and checks that the graphs are isomorphic.

It suppose we should upgrade to the latest sesame. Perhaps worth opening an issue on that if we don't have it.